src/metabase/routes/index.clj

```diff
@@ -66,21 +65,29 @@
   (load-template
    (str "frontend_client/" entrypoint-name ".html")
    (let [{:keys [anon_tracking_enabled google_auth_client_id], :as public-settings} (public-settings/public-settings)]
-     {:bootstrapJSON      (escape-script (json/generate-string public-settings))
+     {:bootstrapJS        (slurp (io/resource "frontend_client/index_inline_bootstrap.js"))
```

Comment:
You should definitely memoize/cache the calls to load these files too so it doesn't have to read the entire file in on every request that loads these pages.

Reply:
Done.